ros2run/ros2run/command/run.py

```diff
@@ -32,6 +32,12 @@ def add_arguments(self, parser, cli_name):
             help="Prefix command, which should go before the executable. "
                  "Command must be wrapped in quotes if it contains spaces "
                  "(e.g. --prefix 'gdb -ex run --args').")
+        try:
+            from argcomplete.completers import SuppressCompleter
+        except ImportError:
+            raise
```

Comment:
Good catch. That was a left over from testing. Fixed in 69ba53a5fd8613ed2a709362ba4f6bb63adf8d0e.